README.md

```diff
@@ -65,6 +66,11 @@ this package to execute our complex `dcp-client` library, which is written in JS
 
 ## Using the library
 
+See also: examples/
+
+### Method 1
+After compiling the project in the `build/src` folder you will find a `.so` file named `pythonmonkey.so`. This is the shared object file that contains the pythonmonkey module.
+
```

Comment:
This is outdated.
We took it out in https://github.com/Distributive-Network/PythonMonkey/commit/2c741ccb4c914f0f2820cd86507c144870514bc9 as the files are now managed by Poetry.

Reply:
```suggestion
```